evaluators/rank/AveragePrecision/__init__.py

```diff
@@ -0,0 +1,38 @@
+from typing import Sequence, Any
+import numpy as np
+from jina.executors.evaluators.rank import BaseRankingEvaluator
+
+
+class AveragePrecisionEvaluator(BaseRankingEvaluator):
```

Comment:
I know it may be design fault ( we will change it) but for now we will need to have a default value of eval at in an __init__ in order to pass our CI.